doc/nrf/ug_thingy91_gsg.rst

```diff
@@ -77,364 +77,403 @@ Complete the following steps to prepare the Thingy:91 for setup:
+   .. group-tab:: Through USB
 
-      The :guilabel:`MCUboot DFU` window appears.
+     To update the modem firmware using USB, complete the following steps:
 
-      .. figure:: /images/thingy91_mcuboot_dfu1.png
-         :alt: Programmer - MCUboot DFU
+      1. Open `nRF Connect for Desktop`_ and launch the Programmer application if you do not have it open already.
+      #. Make sure that :guilabel:`Enable MCUboot` is selected.
+      #. Switch off the Thingy:91.
+      #. Press **SW3** while switching **SW1** to the **ON** position.
 
-         Programmer - MCUboot DFU
+         .. figure:: /images/thingy91_sw1_sw3.svg
+            :alt: Thingy:91 - SW1 SW3 switch
 
-   #. In the :guilabel:`MCUboot DFU` window, click :guilabel:`Write`.
-      When the update is complete, a :guilabel:`Completed successfully` message appears.
-   #. Scroll up in the menu on the left to :guilabel:`File` and click :guilabel:`Clear files`.
+            Thingy:91 - SW1 SW3 switch
 
-   You can now disconnect the Thingy:91 from the computer.
+      #. In the menu, select Thingy:91.
 
-Next, you need to create an nRF Cloud account if you do not have one already.
+      #. In the menu on the left, click :guilabel:`Add file` in the File section, and select :guilabel:`Browse`.
+         A file explorer window appears.
 
-Updating firmware through external debug probe
-==============================================
+         .. figure:: /images/programmer_add_file.png
+            :alt: Programmer - Add file
 
-You can update the Thingy:91 application and modem firmware by using an external debug probe.
+            Programmer - Add file
 
-.. note::
-   The external debug probe must support Arm Cortex-M33, such as the nRF9160 DK.
-   You need a 10-pin 2x5 socket-socket 1.27 mm IDC (Serial Wire Debug (SWD)) JTAG cable to connect to the external debug probe.
+      .. update_modem_start
+
+      5. Navigate to the folder you downloaded and extracted from the `Nordic Semiconductor website`_ in the :ref:`Download firmware <download_firmware_thingy91>` step.
+      #. Find the modem firmware zip file with the name similar to :file:`mfw_nrf9160_*.zip` and the number of the latest version.
 
-To update the firmware, complete the following steps. In these steps, the nRF9160 DK is used as the external debug probe.
+         .. note::
+            Do not extract the modem firmware zip file.
 
-1. Open `nRF Connect for Desktop`_ and launch the Programmer application.
-#. Prepare the hardware:
+      #. Select the zip file and click :guilabel:`Open`.
 
-   a. Connect the Thingy:91 to the debug out port on a 10-pin external debug probe using a JTAG cable.
+      .. update_modem_end
 
-      .. figure:: /images/programmer_thingy91_connect_dk_swd_vddio.svg
-         :alt: Thingy:91 - Connecting external debug probe
+      8. In the Programmer application, scroll down in the menu on the left to :guilabel:`Device` and click :guilabel:`Write`.
```

Comment:
Not sure anything can be done about it, but there's extra space between this and the previous step in the built documentation. 
![image](https://user-images.githubusercontent.com/83582261/174243870-d7fa8601-0fee-42c2-b6d3-e9503f74d0e6.png)

Reply:
I think it is because of the include label.. and I am not aware of any way to circumvent this.